po/zh_CN.po

```diff
@@ -0,0 +1,311 @@
+msgid "Show the CPU monitor"
+msgstr "显示CPU监视器"
+
+#: tophat@fflewddur.github.io/prefs.js:58
+#: tophat@fflewddur.github.io/prefs.js:68
+#: tophat@fflewddur.github.io/prefs.js:82
+msgid "Usage meter"
+msgstr "使用图表"
+
+#: tophat@fflewddur.github.io/prefs.js:59
+#: tophat@fflewddur.github.io/prefs.js:69
+#: tophat@fflewddur.github.io/prefs.js:83
+msgid "Numeric value"
+msgstr "数值"
+
+#: tophat@fflewddur.github.io/prefs.js:60
+#: tophat@fflewddur.github.io/prefs.js:70
+#: tophat@fflewddur.github.io/prefs.js:84
+msgid "Both meter and value"
+msgstr "图表和数值"
+
+#: tophat@fflewddur.github.io/prefs.js:61
+#: tophat@fflewddur.github.io/prefs.js:71
+msgid "Show as"
+msgstr "显示为"
+
+#: tophat@fflewddur.github.io/prefs.js:62
+msgid "Show each core"
+msgstr "显示每个核心"
+
+#: tophat@fflewddur.github.io/prefs.js:65
+msgid "Memory"
+msgstr "内存"
+
+#: tophat@fflewddur.github.io/prefs.js:66
+msgid "Show the memory monitor"
+msgstr "显示内存监视器"
+
+#: tophat@fflewddur.github.io/prefs.js:74
+msgid "Disk"
+msgstr "磁盘"
+
+#: tophat@fflewddur.github.io/prefs.js:75
+msgid "Show the disk monitor"
+msgstr "显示磁盘监视器"
+
+#: tophat@fflewddur.github.io/prefs.js:77
+msgid "Available storage"
+msgstr "可用存储"
+
+#: tophat@fflewddur.github.io/prefs.js:78
+#: tophat@fflewddur.github.io/lib/fs.js:268
+msgid "Disk activity"
+msgstr "磁盘活动"
+
+#: tophat@fflewddur.github.io/prefs.js:79
+msgid "Storage and activity"
+msgstr "存储和活动"
+
+#: tophat@fflewddur.github.io/prefs.js:80
+msgid "Monitor shows"
+msgstr "监视器显示"
+
+#: tophat@fflewddur.github.io/prefs.js:85
+msgid "Show available storage as"
+msgstr "将可用存储显示为"
+
+#: tophat@fflewddur.github.io/prefs.js:92
+msgid "Filesystem to monitor"
+msgstr "要监视的文件系统"
+
+#: tophat@fflewddur.github.io/prefs.js:95
+msgid "Network"
+msgstr "网络"
+
+#: tophat@fflewddur.github.io/prefs.js:96
+msgid "Show the network monitor"
+msgstr "显示网络监视器"
+
+#: tophat@fflewddur.github.io/prefs.js:98
+msgid "Bytes"
+msgstr "字节"
+
+#: tophat@fflewddur.github.io/prefs.js:99
+msgid "Bits"
+msgstr "比特"
+
+#: tophat@fflewddur.github.io/prefs.js:100
+msgid "Measurement unit"
+msgstr "测量单位"
+
+#: tophat@fflewddur.github.io/lib/cpu.js:223
+msgid "Processor usage"
+msgstr "处理器使用情况"
+
+#: tophat@fflewddur.github.io/lib/cpu.js:226
+msgid "Processor utilization:"
+msgstr "处理器利用率："
+
+#: tophat@fflewddur.github.io/lib/cpu.js:254
+#: tophat@fflewddur.github.io/lib/fs.js:303
+#: tophat@fflewddur.github.io/lib/mem.js:272
+#: tophat@fflewddur.github.io/lib/net.js:180
+#, javascript-format
+msgid "%d min ago"
+msgid_plural "%d mins ago"
+msgstr[0] "%d 分钟前"
+msgstr[1] "%d 分钟前"
+
+#: tophat@fflewddur.github.io/lib/cpu.js:257
+#: tophat@fflewddur.github.io/lib/fs.js:306
+#: tophat@fflewddur.github.io/lib/mem.js:275
+#: tophat@fflewddur.github.io/lib/net.js:183
+msgid "now"
+msgstr "现在"
+
+#: tophat@fflewddur.github.io/lib/cpu.js:260
+#: tophat@fflewddur.github.io/lib/fs.js:309
+#: tophat@fflewddur.github.io/lib/mem.js:278
+msgid "Top processes"
+msgstr "Top进程"
```

Comment:
I't better to add a space between Chinese words and English words, just like other part of this PR.

```suggestion
msgstr "Top 进程"
```